### PR TITLE
Views

### PR DESCRIPTION
**Pull request by Roland-Ntwali**
### In this pull request, I did the following:

- Implement the design from the sneak peek wireframes
- Use preprocessed HTML file with embedded Ruby code
- Add user view
- Add post view

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,29 @@
...
+<div class="main_container">
+<div class="profile">
+<div class="photos">
+    <ul class="photo"> 
+   <li> <img src="<%= @users.photo %>"> </li>
+   </ul>
+</div>
+<div class="names_posts">
+    <ul class="name_post"> 
+   <li class="list_name"> <%= @users.name %> </li>
```
Comment (OyePriscilla) [inline app/views/users/show.html.erb]: - [ ] Kindly take note that for good user experience you ought to use the `link `element that will redirects users to the `users_post `page. 👍 I was only able to access the page by typing the `url `manually. ℹ️ 

- [ ] Kindly ensure that `three most recent posts` are displayed by default according to the project requirements. 👍

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,33 @@
...
+              <li class="list_countpost">Number of post: <%= @user.post_counter %> </li>
+           </ul>
+        </div>
+    </div>
+
+    <div class="post">
+        <% @user.posts.each do |post| %>
+        <ul class="post_info width"> 
+            <h3 class="title"><%= post.title%> #<%= post.id%></h3>
+            <li class="bio_content"> <%= post.text %> </li>
```
Comment (OyePriscilla) [inline app/views/posts/index.html.erb]: - [ ] Kindly take note that for good user experience you ought to use the `link` element that will redirects users to the `post_index `page. 👍 I was only able to access the page by typing the `url `manually. ℹ️ 
- [ ] For the sake of semantics, I think the h3 element supposed not to be in the unordered list `ul`. ℹ️

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Roland-Ntwali ✔️ , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
`Users controller #index` action rendered all users ✔️
`recent_post` method to rendered the user's 3 most recent post ❎
Used `post.recent_comments` rendered 5 most recent comments ✔️
`Post show page` displays post with all comments listed ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Zuhaib042 — APPROVED** [review]
Hi @Roland-Ntwali 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Well Done on implementing suggested changes

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**